src/test/java/seedu/address/testutil/TypicalProperties.java

```diff
@@ -34,10 +34,10 @@
 public class TypicalProperties {
 
     public static final Property P_ALICE = new PropertyBuilder()
-            .withName("Jurong West Ave 6")
+            .withName("Jurong West St 62")
             .withPrice(MICROSCOPIC_PRICE)
             .withSeller("Alice Pauline")
-            .withAddress("123, Jurong West Ave 6, #08-111")
+            .withAddress("123, Jurong West St 62, #08-111")
```

Comment:
Just a quick question: whats the point of this change?

Reply:
Another default property uses this address as well, so I changed it to fix the tests